src/chainstate/stacks/index/marf.rs

```diff
@@ -548,7 +608,15 @@ impl<T: MarfTrieId> MARF<T> {
         Ok(result)
     }
 
+    #[cfg(test)]
+    pub fn seal(&mut self) -> Result<TrieHash, Error> {
+        let mut tx = self.begin_tx()?;
+        let h = tx.seal()?;
+        Ok(h)
+    }
+
     // helper method for walking a node's backpr
+    #[inline]
```

Comment:
This is a pretty large method, should we really push for the compiler to inline it?

Reply:
Removed